packages/config/src/providers/dotenv/DotEnvsConfigSource.ts

```
@@ -0,0 +1,40 @@
+
+export type DotEnvsConfigSourceOptions = EnvsConfigSourceOptions & DotenvFlowConfigOptions;
+
+export class DotEnvsConfigSource extends EnvsConfigSource<DotEnvsConfigSourceOptions> {
+  getAll(): Record<string, unknown> {
+    dotenvExpand.expand(dotenv.config(this.options));
+
+    return super.getAll();
+  }
+
+  async watch(onChange?: ConfigSourceOnChangeCB) {
+    const files = await globby(join(this.options.path!, ".env"), {
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)